buildutil.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"go/build"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"golang.org/x/tools/go/vcs"
+)
+
+// forEachRepository calls found for each repository it finds in all GOPATH workspaces.
+func forEachRepository(found func(Repo)) {
+	for _, workspace := range filepath.SplitList(build.Default.GOPATH) {
+		srcRoot := filepath.Join(workspace, "src")
+		if _, err := os.Stat(srcRoot); os.IsNotExist(err) {
+			continue
+		}
+		_ = filepath.Walk(srcRoot, func(path string, fi os.FileInfo, err error) error {
```

Comment:
Is the `_ = filepath.Walk` common to signal that the error is 'intentionally ignored' ? Or just there by accident?